balboa-agent/src/main/resources/config/config.properties

```diff
@@ -4,6 +4,10 @@
 # Current configuration for the Metric Consumer.
 balboa.agent.data.dir = /tmp
 balboa.agent.sleeptime = 10000
+balboa.agent.transport.type = MQ
+# balboa.agent.balboa.http.url
+# balboa.agent.balboa.http.timeout
+# balboa.agent.balboa.http.max.retry.wait
```

Comment:
Can you add `.seconds` to the configuration item names so the units are visible to the reader?